### ISSUE TITLE
training dataset and training learning rate

### ISSUE THREAD

**Issue by RaymondCover** [hqucv/siamban#15]
Hi，I have some problem about the training, I use smaller datasets includes VID and GOT10k. Using 500000 pairs each epoch.  I can get 0.335 EAO in vot2018.
I wanna konw why your lr is smaller than pysot .
I alos hope you cuold offer some information about training datasets' affect.  For example, the effect of gradually expanding the training dataset‘s affect for eao.

**Comment by zeduchen** [comment]
We found that a low learning rate is more stable, you can try to train with a different learning rate.
You can try to use different data sets for training. In the future, we may sort out the impact of different datasets on the training results.

**Comment by RaymondCover** [comment]
Thanks for your help
